pkg/tsdb/cloudmonitoring/time_series_filter_test.go

```diff
@@ -386,6 +386,41 @@ func TestTimeSeriesFilter(t *testing.T) {
 		require.True(t, ok)
 		assert.Equal(t, "114250375703598695", labels["resource.label.instance_id"])
 	})
+
+	t.Run("Parse labels for distribution", func(t *testing.T) {
```

Comment:
```suggestion
t.Run("parseResponse successfully parses metadata for distribution valueType", func(t *testing.T) {
		data, err := loadTestFile("./test-data/3-series-response-distribution-exponential.json")
		require.NoError(t, err)
		assert.Equal(t, 1, len(data.TimeSeries))

		res := &backend.DataResponse{}
		require.NoError(t, (&cloudMonitoringTimeSeriesFilter{}).parseResponse(res, data, "test_query"))

		assert.Equal(t, data2.FrameMeta{
			ExecutedQueryString: "test_query",
			Custom: map[string]interface{}{
				"groupBys":        []string(nil),
				"alignmentPeriod": "",
				"labels": map[string]string{
					"resource.label.project_id": "grafana-prod",
					"resource.type":             "https_lb_rule",
				},
				"perSeriesAligner": "",
			},
		}, *res.Frames[0].Meta)
	})
```

Reply:
I am also suggesting some spaces between the setup, function call being tested, and assertions to make the reading easier.